src/test/kotlin/ecommerce/repository/ProductRepositoryTest.kt

```diff
@@ -0,0 +1,50 @@
+package ecommerce.repository
+
+import ecommerce.model.Option
+import ecommerce.model.Product
+import org.assertj.core.api.Assertions.assertThat
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import kotlin.test.Test
+
+@DataJpaTest
+class ProductRepositoryTest {
+    @Autowired
+    private lateinit var productJpaRepository: ProductJpaRepository
+
+    @Test
+    fun save() {
+        val option = Option("test", 10)
+        val expected = Product("test", mutableListOf(option), 10.0, "https://google.com")
+        val actual = productJpaRepository.save(expected)
+        assertThat(actual.id).isNotNull
+        assertThat(actual.id).isNotZero
+        assertThat(actual.name).isEqualTo(expected.name)
+    }
```

Comment:
Nice learning test! 👍 

To make it even more better, why don't we assert option as well?
Since we used a special data for option, it would be great to check if that value was also persisted as well :)